### DISCUSSION TITLE
Manpager vim settings: number not working after jump

### DISCUSSION THREAD

**Discussion by tipa40** [neovim/neovim#27786] [Q&A] [answered]
Hi,
I tried export MANPAGER='nvim +Man! "+set scrolloff=999 relativenumber number"'
which works well initially. I then jumped using a link using ^] 
On the second, jumped-to, page the number and relativenumber part is gone, but the scrolloff is still working.

Is there a way I can get it to work? (I am currently using NVIM v0.10.0-dev-2363+gb76a01055)
Can I post an issue about it?

**Comment by zeertzjq** [comment, marked as answer]
It's better to add `setlocal scrolloff=999 relativenumber number` to `~/.config/nvim/after/ftplugin/man.vim`.

**Reply by tipa40** [reply]
Nice, thank you so much! 😊👍🏻
